scripts/bulk-password-change.js

```diff
@@ -40,25 +42,30 @@ const options = {
   json: true,
   headers: {
     'Authorization': 'Basic ' + Buffer.from(`${user}:${password}`).toString('base64')
-  }   
+  }
 };
 
 const execute = async () => {
   let users = [];
   try {
-    users = await rpn.get(options);
+    const getOptions = Object.assign({ qs: { include_docs: true, startkey:  '"org.couchdb.user"' } }, options);
+    users = await rpn.get(getOptions);
   } catch (e) {
     console.log('An error while getting the list of users - ', e.message);
   }
-  
-  users.forEach(async (user) => {
+
+  users.rows.forEach(async (user) => {
     const postOptions = Object.assign({}, options);
     postOptions.body = {
-      'password': 'Secret_1'
+      _rev: user.doc._rev,
+      name: user.doc.name,
+      roles: user.doc.roles,
+      type: user.doc.type,
+      password: 'Secret_1'
```

Comment:
I think we can be less presumptuous here.  Instead of hard coding what fields we think are on the user that there might be, we can just do something this:

```
postOptions.body = user.doc;
postOptions.body.password = 'Secret_1';
````

this way we don't miss future fields that might get added.  You can check out [how I did this in a near identical script](https://github.com/medic/chis-interoperability/blob/cht-set-external-id/cht-config/scripts/set-external-id.js), and then just added a `'?rev=' + user.doc._rev` to the URL so it knows which revision to reference.

This is the only code specific thing I see, I'll dig into the docs and testing the instructions tomorrow!